router/pkg/config/config.go

```diff
@@ -124,11 +124,15 @@ type TrafficShapingRules struct {
 	Router RouterTrafficConfiguration `yaml:"router"`
 }
 
+type FileUploadsRules struct {
+	Enabled          bool        `yaml:"enabled" default:"true"`
```

Comment:
Please provide env support for these configs `file_upload_enabled`

Reply:
Done